devicetypes/smartthings/zwave-lock-without-codes.src/zwave-lock-without-codes.groovy

```diff
@@ -23,6 +23,7 @@ metadata {
 		capability "Health Check"
 		capability "Configuration"
 
+		fingerprint inClusters: "0x62, 0x71, 0x80, 0x85"
```

Comment:
How does this work exactly? If a device supports one or more of these command classes, then it gets assigned this DTH?